colors/everforest.vim

```diff
@@ -1324,6 +1324,7 @@ call everforest#highlight('InclineNormalNC', s:palette.grey1, s:palette.bg2)
 " }}}
 " echasnovski/mini.nvim {{{
 call everforest#highlight('MiniAnimateCursor', s:palette.none, s:palette.none, 'reverse,nocombine')
+call everforest#highlight('MiniFilesFile', s:palette.fg, s:palette.none)
```

Comment:
Why move highlights so far away from where they were originally defined?

Reply:
I followed the established convention in the Mini section where highlight groups that are defined via `call everforest#highlight` are all done before those that are simply defined via `highlight! link`. In each respective section, the highlight groups are ordered alphabetically, which I also honored.

For all of these PRs, I modified two highlight groups: `MiniFilesFile` and `MiniPickPrompt`. Both of which used to be defined as links, but now are explicitly defined. As such, I had to move their definitions from the "link" section to the "call" section.

Reply:
Alright fair enough.
There is no clearly established convention regarding declaring function calls before links, and I think we should instead keep related highlights _together_ regardless of whether they are links or not. But since things have been organized that way for mini.nvim, let's stick with that structure here 👍